src/null-safety/index.md

```diff
@@ -160,8 +157,10 @@ doesn't depend on Dart 3 breaking changes:
 
 ## Dart 2.x and null safety {#enable-null-safety}
 
-In Dart 2.12 to 2.19, null safety is a configuration option in the pubspec. Null
-safety is not available in SDK versions prior to Dart 2.12.
+From Dart 2.12 to 2.19, you need to enable null safety.
+To enable null safety, set the `sdk` option to `'>=2.12.0'`
+in the `pubspec.yaml` file.
```

Comment:
Remove these sentences, the next few sentences (L166-169) go into greater detail about it already.

```suggestion
```